src/redux/actions/projects/loadProjects.js

```diff
@@ -39,27 +34,10 @@ const loadProjects = () => async (dispatch) => {
     type: PROJECTS_LOADING,
   });
 
-  let url;
-
   try {
-    let data;
-
-    if (config.currentApiVersion === api.V1) {
-      url = '/v1/projects';
-      data = await fetchAPI(url);
-
-      // filter out "projects" that are actually old experiments without a project
-      data = data.filter((project) => project.name !== project.uuid);
-
-      await Promise.all(data
-        .filter((entry) => entry.samples.length)
-        .map((entry) => dispatch(loadSamples(null, entry.uuid))));
-    } else if (config.currentApiVersion === api.V2) {
-      url = '/v2/experiments';
-      data = await fetchAPI(url);
+    let data = await fetchAPI('/v2/experiments');
 
-      data = toApiV1(data);
-    }
+    data = toApiV1(data);
```

Comment:
So we are going to have this forever? Are we planning to remodel redux and not have these toV1 functions but instead just use the unmodified data?

Reply:
No, when we remove project its going away